### PR TITLE
feat(logs): opensearch -> elasticsearch

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Fixes NAN-1000

Due do lack of tooling, outdated sdk and minimal cloud support, we have decided to use elasticsearch instead of opensearch. 

- Use elasticsearch 
I haven't changed the env var to avoid creating more trouble but not sure if it's a good idea. Let me know

- Disabled elasticsearch in self-hosted docker compose
Decision was made to have console log and empty state in the UI if this feature is disabled. So that's it's not required in the critical path.

### CODE REVIEW

**Review by khaliqgant — APPROVED** [review]
> I haven't changed the env var to avoid creating more trouble but not sure if it's a good idea. Let me know

Definitely think its worth changing the env var

**Comment by TBonnin** [comment]
> > I haven't changed the env var to avoid creating more trouble but not sure if it's a good idea. Let me know
> 
> Definitely think its worth changing the env var

I second that

File: docker-compose.yaml [hunk 1]
```diff
@@ -58,31 +59,30 @@ services:
...
+    #     volumes:
+    #         - elasticsearch-data1:/usr/share/elasticsearch/data
+    #     ports:
+    #         - 9500:9200
+    #         - 9600:9300
+    #     environment:
+    #         - discovery.type=single-node
+    #         - xpack.security.enabled=false
+    #     networks:
+    #         - nango
```
Comment (TBonnin) [inline docker-compose.yaml:81]: commented out?

**Review by TBonnin — APPROVED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (bodinsamuel) [reply docker-compose.yaml:81]: The main issue we had in slack community is people not willing/understanding why it was loaded by default. Especially in oauth mode it's not that useful, so we decided to have a screen in the UI that tell how to enable this (aka uncommenting)

**Comment by bodinsamuel** [comment]
> Definitely think its worth changing the env var 

Renamed ☺️

**Pull request merged**